deploy/helm/prometheus-overrides.yaml

```diff
@@ -49,39 +49,53 @@ prometheus:
       matchLabels:
         app: fluent-bit
   prometheusSpec:
+    thanos:
+      version: v0.10.0
+    containers:
+    - name: "prometheus-config-reloader"
+      env:
+      - name: CHART
+        valueFrom:
+          configMapKeyRef:
+            name: sumologic-configmap
```

Comment:
Have we tested the non-helm case? What happens if the configmap has not been created yet at the time of prometheus install? Will this be able to pick up the values from the configmap later when it is created? Same for `fluent-bit`

Reply:
Since the configmap creation is part of the `fluentd-sumologic.yaml.tmpl`, it will be created as the first step as per the Non-Helm Installation docs, so it will be available by default. https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v0.14.0/deploy/docs/Non_Helm_Installation.md#deploy-fluentd